huxley/www/js/components/AdvisorRosterView.js

```diff
@@ -257,10 +257,12 @@ class AdvisorRosterView extends React.Component {
                 <Button
                   color="yellow"
                   size="small"
-                  onClick={this._handleDelegatePasswordChange.bind(
-                    this,
-                    delegate
-                  )}
+                  onClick={e => 
+                    window.confirm(`Are you sure you wish to reset the password for ${delegate.name}? The delegate will no longer be able to log in with the old password. Please ensure that the delegate's email is entered correctly: ${delegate.email}`) 
+                    && this._handleDelegatePasswordChange.bind(
```

Comment:
You don't need to bind anymore, that was to create a new function with the arguments already bound. Now that we're in a lambda, just call the function normally.

Reply:
Ah I see, thanks! Should we replace all binds with lambdas in general? I guess this code was written prior to lambdas being a thing?

Seems this stack post recommends lambdas over binds but maybe theres a reason to leave them be?
https://stackoverflow.com/questions/42117911/lambda-functions-vs-bind-memory-and-performance